### ISSUE TITLE
[Truffle] exec(env, *command) gives ENOENT

### ISSUE THREAD

**Issue by eregon** [jruby/jruby#4357]
### Expected Behavior

```ruby
exec({"FOO" => "BAR"}, "bash", "-c", "echo $FOO")
=> BAR
```

### Actual Behavior

```
-e:1:in `exec': No such file or directory (Errno::ENOENT)
	from -e:1:in `<main>'
```

From #4346.

**Comment by eregon** [comment]
Related: we could use a native `execve` and share logic that `#spawn` uses for all the flags.

**Comment by nirvdrum** [comment]
Fixed in 0f0b9522ab98c8fe8f85ccb46c6eac0c80e5e173.
